docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -638,7 +636,271 @@ Take the following code for example.
+   +   if local:
+   +       # Local: configure local distributed training.
+   +       dist.init_process_group(backend="gloo|nccl")
+   +       os.environ["USE_TORCH_DISTRIBUTED"] = "true"
+   +       distributed_context = core.DistributedContext.from_torch_distributed()
+   +       latest_checkpoint = None
+   +   else:
+   +       # On-cluster: Determined will automatically detect distributed context.
+   +       distributed_context = None
+   +       # On-cluster: configure the latest checkpoint for pause/resume training functionality.
+   +       latest_checkpoint = det.get_cluster_info().latest_checkpoint
+
+   +     with det.pytorch.init(
+   +       distributed=distributed_context
+         ) as train_context:
+             trial = MNistTrial(train_context)
+             trainer = det.pytorch.Trainer(trial, train_context)
+             trainer.fit(
+                 max_length=pytorch.Epoch(1),
+                 checkpoint_period=pytorch.Batch(100),
+                 validation_period=pytorch.Batch(100),
+   +             latest_checkpoint=latest_checkpoint,
+             )
+
+To run Trainer API solely on-cluster, the code is much simpler:
+
+.. code:: python
+
+   def main():
+       with det.pytorch.init() as train_context:
+           trial_inst = model.MNistTrial(train_context)
+           trainer = det.pytorch.Trainer(trial_inst, train_context)
+           trainer.fit(
+               checkpoint_period=pytorch.Batch(100),
+               validation_period=pytorch.Batch(100),
+               latest_checkpoint=det.get_cluster_info().latest_checkpoint,
+           )
+
+Submit Your Trial for Training on Cluster
+=========================================
+
+To run your experiment on cluster, you'll need to create an experiment configuration (YAML) file.
+Your experiment configuration file must contain searcher configuration and entrypoint.
+
+.. note::
+
+   ``hyperparameters.global_batch_size`` is required if ``max_length`` is configured in records
+
+.. code:: python
+
+   name: pytorch_trainer_trial
+   searcher:
+     name: single
+     metric: validation_loss
+     max_length:
+       epochs: 1
+   resources:
+     slots_per_trial: 8
+   entrypoint: python3 -m determined.launch.torch_distributed python3 train.py
+
+Submit the trial to the cluster:
+
+.. code:: bash
+
+   det e create det.yaml .
+
+If your training code needs to read some values from the experiment configuration,
+``pytorch.init()`` accepts an ``exp_conf`` argument which allows calling
+``context.get_experiment_config()`` from ``PyTorchTrialContext``.
+
+Loading Checkpoints
+===================
+
+To load a checkpoint from a checkpoint saved using Trainer, you'll need to download the checkpoint
+to a file directory and use an import helper method to import modules. You should instantiate your
+loaded Trial with a ``CheckpointLoadContext``.
+
+``det.import_from_path`` allows you to import from a specific directory and cleans up afterwards.
```

Comment:
@tara-det-ai I was too rushed to put the `det.import_from_path` docstring in the docs (I was in a hurry to unblock Trainer API).  But where do you think we should put it?  It's sort of a framework-agnostic utility.

I can make a PR to add it somewhere, so Anda can link to it here rather than copy/paste parts of the `import_from_path` docstring into this document.

Reply:
we'll add it to the PyTorch Reference page with a note to let users know it is a framework-agnostic utility; if we have many of these framework-agnostic utilities that we've written we can then make an index page for them